sktime/base/_base.py

```diff
@@ -672,5 +785,13 @@ def deepcopy_func(f, name=None):
     )
 
 
+def _rm_underscore(x):
```

Comment:
should be replaced by `str.rstrip` - suggestion courtesy of @tpvasconcelos, see https://github.com/sktime/skbase/pull/293

Reply:
fixed, thanks!